online-norm/tensorflow/online_norm_tf/online_norm.py

```diff
@@ -809,9 +792,13 @@ def conv_alongb_w1(input, b=self.b_size, c=self.ch):
             c_input = tf.transpose(tf.reshape(tf.transpose(input,
                                                            perm=[1, 0, 2]),
                                               [2 * b, -1]))
-            out = tf.nn.conv1d(tf.expand_dims(c_input, 2),
-                               tf.constant([1.], dtype=self.mp_type, shape=[b, 1, 1]),
-                               stride=1, padding='VALID', data_format="NWC")
+            out = tf.nn.conv1d(
+                tf.expand_dims(c_input, 2),
+                tf.constant([1.], dtype=self.mp_type, shape=[b, 1, 1]),
+                stride=1,
+                padding='VALID',
+                data_format="NWC"
```

Comment:
shouldn't it be NCW?

Reply:
What I have is correct, the BON tests verify that.